sdk/textanalytics/platform-matrix-ci.json

```diff
@@ -0,0 +1,13 @@
+{
+  "matrix": {
+    "$IMPORT": "eng/pipelines/templates/stages/platform-matrix.json"
+  },
+  "include": [
+    {
+      "Agent": { "ubuntu-20.04": { "OSVmImage": "MMSUbuntu20.04", "Pool": "azsdk-pool-mms-ubuntu-2004-general" } },
```

Comment:
should this be on 22.04 at this point?

Reply:
We are still using the 20.04 in our repo. Will need to create another PR to upgrade all if needed.